Documentation/Contributing/development-environment.md

```diff
@@ -58,3 +58,37 @@ Use [helm.sh](https://github.com/rook/rook/blob/master/tests/scripts/helm.sh) to
 
 !!! note
     If Helm is not available in your `PATH`, Helm will be downloaded to a temporary directory (`/tmp/rook-tests-scripts-helm`) and used from that directory.
+
+
+## Using local Rook image on minikube cluster
+
+Developers can test quickly their changes by building and using the local Rook image
+on their minikube cluster as following:
+
+1) Set minikube Docker environment variables:
```

Comment:
Why is this step neede?

Reply:
Otherwise `kubectl` will use your local docker setup (instead of the minikube's). @travisn  please, can you double check just in case I'm mistaken.

Reply:
Yes this is correct, this is how you can build from your laptop console to use docker inside minikube. Then no need to move the image into minikube after building.
```suggestion
1) Set the local Docker environment to use minikube:
```